### DISCUSSION TITLE
Struggling to get the correct MS Token

### DISCUSSION THREAD

**Discussion by annashamsili** [davidteather/TikTok-Api#1115] [Q&A] [answered]
I'm trying to run the basic example provided to test the API. I went on TikTok -> developer tools and have tried using many different types of MS token. In most cases, it just returns nothing, and twice it gave me **EmptyResponseException(result, "TikTok returned an empty response")**. 

I think I'm struggling to see which MS token is from "my" cookies on TikTok - the msToken in storage -> cookies changes every 10-ish seconds, so I'm not sure if it's that one but I'm doing something wrong as it keeps changing? The other tokens I tried were in the "network" tab, and as said most of the time it runs the script and nothing happens. 

This is the example:

```
from TikTokApi import TikTokApi
import asyncio
import os

ms_token = os.environ.get("ms_token", None) # get your own ms_token from your cookies on tiktok.com

async def trending_videos():
    async with TikTokApi() as api:
        await api.create_sessions(ms_tokens=[ms_token], num_sessions=1, sleep_after=3)
        async for video in api.trending.videos(count=30):
            print(video)
            print(video.as_dict)

if __name__ == "__main__":
    asyncio.run(trending_videos())
```

I can see other questions in similar vein but most seem to say find the token in developer tools - which I am, but I'm not sure if I'm grabbing the wrong one or doing something else wrong?

**Comment by nikolaevaxenov** [comment, marked as answer]
Try to replace
```python
await api.create_sessions(ms_tokens=[ms_token], num_sessions=1, sleep_after=3)
```
with 
```python
await api.create_sessions(headless=False, ms_tokens=[ms_token], num_sessions=1, sleep_after=3)
```

**Reply by annashamsili** [reply]
This worked, thank you!
